tools/tts/tts-api/tts-api.py

```diff
@@ -18,18 +17,20 @@ def hhmmss_to_seconds(string):
 	new_time += float(separated_times[2])
 	return new_time
 
-def text_to_speech_handler(endpoint, voice, text, filter_complex):
+def text_to_speech_handler(endpoint, voice, text, filter_complex, pitch, silicon = False):
 	filter_complex = filter_complex.replace("\"", "")
-
-	response = requests.get(f"http://tts-container:5003/" + endpoint, json={ 'text': text, 'voice': voice })
+	response = requests.get(f"http://tts-container:5003/" + endpoint, json={ 'text': text, 'voice': voice, 'pitch': pitch })
 	if response.status_code != 200:
 		abort(500)
 
 	ffmpeg_result = None
 	if filter_complex != "":
 		ffmpeg_result = subprocess.run(["ffmpeg", "-f", "wav", "-i", "pipe:0", "-filter_complex", filter_complex, "-c:a", "libvorbis", "-b:a", "64k", "-f", "ogg", "pipe:1"], input=response.content, capture_output = True)
 	else:
-		ffmpeg_result = subprocess.run(["ffmpeg", "-f", "wav", "-i", "pipe:0", "-c:a", "libvorbis", "-b:a", "64k", "-f", "ogg", "pipe:1"], input=response.content, capture_output = True)
+		if silicon:
+			ffmpeg_result = subprocess.run(["ffmpeg", "-f", "wav", "-i", "pipe:0", "-i", "./SynthImpulse.wav", "-i", "./RoomImpulse.wav", "-filter_complex", "[0] aresample=44100 [re_1]; [re_1] apad=pad_dur=2 [in_1]; [in_1] asplit=2 [in_1_1] [in_1_2]; [in_1_1] [1] afir=dry=10:wet=10 [reverb_1]; [in_1_2] [reverb_1] amix=inputs=2:weights=8 1 [mix_1]; [mix_1] asplit=2 [mix_1_1] [mix_1_2]; [mix_1_1] [2] afir=dry=1:wet=1 [reverb_2]; [mix_1_2] [reverb_2] amix=inputs=2:weights=10 1 [mix_2]; [mix_2] equalizer=f=7710:t=q:w=0.6:g=-6,equalizer=f=33:t=q:w=0.44:g=-10 [out]; [out] alimiter=level_in=1:level_out=1:limit=0.5:attack=5:release=20:level=disabled", "-c:a", "libvorbis", "-b:a", "64k", "-f", "ogg", "pipe:1"], input=response.content, capture_output = True)
```

Comment:
would've preferred if you did some modular way of doing this so that other filter effects like this could easily be implemented instead of hardcoding them directly into a python file.

Reply:
maybe down the line, this entire setup could use a refactor but not right now i'm way too tired